includes/class-wc-google-analytics.php

```diff
@@ -350,7 +350,7 @@ protected function get_ecommerce_tracking_code( $order_id ) {
 	 * @return bool True if tracking for a certain setting is disabled
 	 */
 	private function disable_tracking( $type ) {
-		if ( is_admin() || current_user_can( 'manage_options' ) || ( ! $this->ga_id ) || 'no' === $type || apply_filters( 'woocommerce_ga_disable_tracking', false, $type ) ) {
+		if ( current_user_can( 'manage_options' ) || ( ! $this->ga_id ) || 'no' === $type || apply_filters( 'woocommerce_ga_disable_tracking', false, $type ) ) {
```

Comment:
Instead of removing that perhaps check if its doing AJAX? `if ( ( ! wp_doing_ajax() && is_admin() ) || ...`